test/cqlpy/test_materialized_view.py

```diff
@@ -1360,11 +1358,17 @@ def test_create_materialized_view_invalid_char_name(cql, test_keyspace, table1):
         # We shouldn't reach here, but if we did, let the test fail cleanly
         cql.execute(f'DROP MATERIALIZED VIEW IF EXISTS {test_keyspace}."xyz!123"')
 
+# Verify that zero-sized materialized # view names are cleanly rejected,
+# as InvalidRequest or ConfigurationException.def test_create_materialized_view_empty_name(cql, test_keyspace, table1):
```

Comment:
This comment has some garbage (like "#" and "def test_create_...") and also mentions the wrong exception type. Please fix it.

Reply:
done